index.html

```diff
@@ -1,271 +1,511 @@
+            class="btn weight-500 size-17 line_hieght-24 clr-p-500 background-white"
+          >
+            Get my resume
+          </button>
+        </div>
+        <ul class="tools ul">
+          <li class="langs">
+            <div class="langs__top flex">
+              <h3
+                class="langs__top__title weight-500 size-20 line_hieght-24 clr-n-900"
+              >
+                Languages
+              </h3>
+              <img
+                class="langs__top__img"
+                src="img/arrow.png"
+                alt="arrow icon"
+              />
+            </div>
+            <ul class="langs__bottom ul">
+              <li class="flex bg-n-300">
+                <img src="img/js.png" alt="javascript icon" />
+                <p
+                  class="langs__bottom__name weight-500 size-15 line_hieght-20 clr-n-700"
+                >
+                  JavaScript
+                </p>
+              </li>
+              <li class="flex bg-n-300">
+                <img src="img/html.png" alt="html icon" />
+                <p
+                  class="langs__bottom__name weight-500 size-15 line_hieght-20 clr-n-700"
+                >
+                  HTML
+                </p>
+              </li>
+              <li class="flex bg-n-300">
+                <img src="img/css.png" alt="css icon" />
+                <p
+                  class="langs__bottom__name weight-500 size-15 line_hieght-20 clr-n-700"
+                >
+                  CSS
+                </p>
+              </li>
+            </ul>
+          </li>
+          <li class="about__frameworks">
+            <div class="about__frameworks__top flex">
+              <h3
+                class="about__frameworks__top__title weight-500 size-20 line_hieght-24 clr-n-900"
+              >
+                Frameworks
+              </h3>
+              <img
+                class="about__frameworks__top__img"
+                src="img/arrow.png"
+                alt="arrow icon"
+              />
+            </div>
+          </li>
+          <li class="about__skills">
+            <div class="about__skills__top flex">
+              <h3
+                class="about__skills__top__title weight-500 size-20 line_hieght-24 clr-n-900"
+              >
+                Skills
+              </h3>
+              <img
+                class="about__skills__top__img"
+                src="img/arrow.png"
+                alt="arrow icon"
+              />
+            </div>
+          </li>
+        </ul>
+      </section>
+    </div>
+
+    <!-- From ensaha -->
+    <section class="form-container-2">
+      <strong class="form-header-2">Contact me</strong>
+      <div class="contact-form-2">
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I'd love to help
+          with it!
+        </p>
+        <form class="contact-me-2" action="">

```

Comment:
Kindly ensure that the form uses the action and method attributes to integrate with [formspree service](https://formspree.io/html). This is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/91668f1d04ab043c8fac118a333ea1d5695b54b3/portfolio/3_contact_form.md?plain=1#L48) for the project.

In order to let formspree recognize each input field, you may have to specify a name attribute for each of them.

Reply:
> Kindly ensure that the form uses the action and method attributes to integrate with [formspree service](https://formspree.io/html). This is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/91668f1d04ab043c8fac118a333ea1d5695b54b3/portfolio/3_contact_form.md?plain=1#L48) for the project.
> 
> In order to let formspree recognize each input field, you may have to specify a name attribute for each of them.

🆗 I am sorry i forgot it i will add wait a moment.